amical/data_processing.py

```diff
@@ -376,6 +374,7 @@ def show_clean_params(
     offy=0,
     apod=False,
     window=None,
+    ifu=False,
     *,
```

Comment:
New arguments should always go after `*,`
```suggestion
    *,
    ifu=False,
```